etc/config-template.toml

```diff
@@ -155,6 +155,9 @@ stats-dump-period-sec = 600
 # least 2MB;
 # compaction_readahead_size = 0
 
+# Limit the disk IO of compaction and flush.
+# rate-bytes-per-sec = "100MB"
```

Comment:
I don't think we should leave "100MB" here, it may make people think that's the default settings.